### ISSUE TITLE
Ability to auto-calculate skill totals

### ISSUE THREAD

**Issue by Mottokrosh** [Mottokrosh/SheetIssues#2]
Turn equal sign into button to accomplish this?

**Comment by PGarrano** [comment]
Most total signs could be turned to buttons to auto add everything: useful for skills, AC, saving throws, etc.

**Comment by Mottokrosh** [comment]
Indeed. I'll have to figure out a way to handle conditional stuff that a user might have put in the fields. I often put stuff in brackets in there for example, so I should probably ignore values in brackets.

**Comment by PGarrano** [comment]
I've seen some excel sheets that had a nifty solution: say you had a +4 bonus to acrobatics, with +2 when jumping, it would print out +4 acrobatics (+6 when jumping).
A simpler solution might be simply not to add conditional bonuses, just keep them referenced somewhere.

**Comment by ninjadwarf** [comment]
If you ensure there is both a misc and a conditional field for anything that will be auto summed, then the misc can be part of the total and the conditional can be used for player to consider.

**Comment by GeoGranpa** [comment]
You cannot know the current game conditions through the character sheet. You can only assume that the user is writing the bonuses that will be applied always and auto-calculate the total.

Also will be awesome have one column showing the total of the armor penalty check. This value must be taken in consideration when you get the skill total value.

About training bonus, you can parse the input data to check empty values or values less than 1. The problem here is identify if the value introduced is a number or not. I think that you can solve this with a strong regular expression.

I hope this will help with your development.
